mmv1/templates/terraform/examples/filestore_instance_full.tf.erb

```diff
@@ -28,3 +28,15 @@ resource "google_filestore_instance" "<%= ctx[:primary_resource_id] %>" {
     connect_mode = "DIRECT_PEERING"
   }
 }
+
+resource "google_filestore_backup" "backup" {
```

Comment:
This example's intention is to show the capabilities of Filestore instance, not Filestore Backup. We should remove changes to this example. From what I can tell, the backup part of this example doesn't test anything different than `mmv1/templates/terraform/examples/filestore_backup_basic.tf.erb`